webknossos-datastore/app/com/scalableminds/webknossos/datastore/datareaders/zarr/NgffMetadata.scala

```diff
@@ -57,6 +58,22 @@ object NgffMultiscalesItem {
   implicit val jsonFormat: OFormat[NgffMultiscalesItem] = Json.format[NgffMultiscalesItem]
 }
 
+case class NgffMultiscalesItemV2(
```

Comment:
I’d say the v2 stuff should be in separate file. If some code is shared between v1 and v2, maybe add a third file for the shared stuff.

Also, is v2 the right version number? I thought they were like at 0.5?

Also, please add some links here to the spec documents.